vagrant/roles/sit.glusterfs/tasks/setup/centos8.yml

```
@@ -0,0 +1,9 @@
+---
+- name: Process common OS tasks
+  include_tasks: "{{ config.os[config.nodes[inventory_hostname].os].distro }}.yml"
```

This file already has knowledge about the OS and version. It doesn't need to use `os` and `distro`. It can directly use the centos.yml file.

```suggestion
  include_tasks: centos.yml
```